src/app/header.js

```diff
@@ -136,6 +137,7 @@ export class Header extends EventSubscriber {
              let url =
                 Misc.assembleImageLink(
                     this.context.server,
+                    this.context.getPrefixedURI(WEBCLIENT),
```

Comment:
internally getPrefixedURI uses Misc.prepareURI which 'prepares' it such that I have slash in the beginning and no trailing slash